network_wrangler/TransitNetwork.py

```diff
@@ -646,15 +648,27 @@ def _apply_transit_feature_change_routing(
         if stops_change:
             # If node IDs in properties["set_stops"] are not already
             # in stops.txt, create a new stop_id for them in stops
-            if any(
-                x not in stops[TransitNetwork.STOPS_FOREIGN_KEY].tolist()
-                for x in properties["set_stops"]
-            ):
-                # FIXME
-                WranglerLogger.error(
-                    "Node ID is used that does not have an existing stop."
-                )
-                raise ValueError
+            existing_fk_ids = set(
+                stops[TransitNetwork.STOPS_FOREIGN_KEY].tolist()
+            )
+            nodes_df = self.road_net.nodes_df.loc[
+                :, [TransitNetwork.STOPS_FOREIGN_KEY, "x", "y"]
+            ]
+            for fk_i in properties["set_stops"]:
+                if fk_i not in existing_fk_ids:
+                    WranglerLogger.info(
+                        "Creating a new stop in stops.txt for node ID: {}".format(fk_i)
+                    )
+                    # Add new row to stops
+                    stops.loc[len(stops.index) + 1, [
+                        "stop_id", "stop_lat", "stop_lon",
+                        TransitNetwork.STOPS_FOREIGN_KEY
+                    ]] = [
+                        str(int(fk_i) + TransitNetwork.STOP_ID_SCALAR),
```

Comment:
We would need to verify that whatever we use for `stop_id` is unique (either `fk_i` *or* `fk_i` + a scalar )

Reply:
Done.